ScriptHookVDotNet.sln

```diff
@@ -1,8 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.40629.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2013 or higher
```

Comment:
This line is responsible for the solution icon displayed in the explorer. "Visual Studio 2013" adds a "12" badge, "Visual Studio 14" adds "14" badge. Unfortunately, this one removes that, so I'd remove the " or higher" part.